README.md

```diff
@@ -83,6 +84,31 @@ pipelines:
 ```
 
 
+### Example
```

Comment:
I like that you've provided an example :+1: As a small addition, it would be good to explain the ordering and key columns that are used.

Reply:
A detailed description of ordering and key columns is already there in the configuration table above. I have added a brief description in the example also.